plugins/org.jboss.ide.eclipse.freemarker/src/org/jboss/ide/eclipse/freemarker/model/interpolation/BuiltInInfo.java

```diff
@@ -1,13 +1,13 @@
-/******************************************************************************* 
- * Copyright (c) 2015 Red Hat, Inc. 
- * Distributed under license by Red Hat, Inc. All rights reserved. 
- * This program is made available under the terms of the 
- * Eclipse Public License v1.0 which accompanies this distribution, 
- * and is available at http://www.eclipse.org/legal/epl-v10.html 
- * 
- * Contributors: 
- * Red Hat, Inc. - initial API and implementation 
- ******************************************************************************/ 
+/*******************************************************************************
```

Comment:
Can you remove changes that are white-space related (I assume the IDE did them automatically)?

Reply:
The IDE did remove unnecessary whitespace at the end.
The best would be to include a formatter - perhaps in the project?

For the other remark, as said in the other PR, I think this is wrong: you still get a warning:

> juil. 09, 2021 5:56:33 PM freemarker.log._JULLoggerFactory$JULLogger error
GRAVE: DefaultObjectWrapper.incompatibleImprovements was set to the object returned by Configuration.getVersion(). That defeats the purpose of incompatibleImprovements, and makes upgrading FreeMarker a potentially breaking change. Also, this probably won't be allowed starting from 2.4.0. Instead, set incompatibleImprovements to the highest concrete version that's known to be compatible with your application.
> juil. 09, 2021 5:56:33 PM freemarker.log._JULLoggerFactory$JULLogger error
GRAVE: Configuration.incompatibleImprovements was set to the object returned by Configuration.getVersion(). That defeats the purpose of incompatibleImprovements, and makes upgrading FreeMarker a potentially breaking change. Also, this probably won't be allowed starting from 2.4.0. Instead, set incompatibleImprovements to the highest concrete version that's known to be compatible with your application.

My understanding is that you should always use the 2_3_XX constant rather than doing that.

That said, I reverted this modification.

Reply:
Autoformatting, probably I should do it. But then that's separate from this PR.

I swear the version trick works. I have just tried it, and it does not log anything:

        Version lastVersion = new Version(Configuration.getVersion().toString());
        Configuration cfg = new Configuration(lastVersion);

Anyway, I can deal with that separately.